lib/publisher.ex

```diff
@@ -32,13 +32,19 @@ defmodule GenRMQ.Publisher do
 
   `app_id` - publishing application ID
 
+  `activate_confirmations` - activates publishing confirmations on the channel
```

Comment:
How about `enable_confirmations`? And the default is `false` right?

Reply:
Sounds good. Yes, `false` is default.